azure-pipelines.yml

```diff
@@ -89,7 +89,7 @@ stages:
       - job: compatibility_matrix
         condition: or(eq(variables.isScheduled, true), eq(variables.isMain, true))
         variables:
-          GE_pytest_opts: '--no-sqlalchemy'
+          GE_pytest_opts: '--no-sqlalchemy --spark'
```

Comment:
This should be `--no-spark` right?

Reply:
Changes requested just to block auto merge - let's confirm which flags are needed for which steps.